src/agent/src/pci.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright Red Hat.
```

Comment:
nit: missing year.

Reply:
As noted somewhere else, the Red Hat guidelines specifically suggest just "Copyright Red Hat".  "(c)" is redundant with the word Copyright, and the year is apparently unnecessary and usually out of date anyway.